jerry-core/parser/js/js-lexer.c

```diff
@@ -1163,7 +1167,7 @@ lexer_parse_string (parser_context_t *context_p, /**< context */
        * after a backslash). Always converted to two 3 byte
        * long sequence. */
       length += 2 * 3;
-      has_escape = true;
+      status_flags |= LEXER_LIT_LOCATION_HAS_ESCAPE;
```

Comment:
Is'n this flag is set already (and is_ascii is set to false)? I think both LEXER_LIT_LOCATION_HAS_ESCAPE and LEXER_LIT_LOCATION_IS_ASCII cannot be set at the same time. Maybe the old code was unnecessary.

Reply:
If i remove this flag, the lit_is_valid_cesu8_string function will return false and it will cause an assert within ecma_new_ecma_string_from_utf8 function. (For example, when i ran this test: assert(JSON.stringify("ab𬄕c") === '"ab𬄕\\u001fc"'); ) I think, we need to keep this one.

Reply:
I think this should be also `=`